src/components/Game.vue

```diff
@@ -329,7 +344,7 @@ export default {
             KEYBOARD_QWERTY,
             KEYBOARD_QWERTZ,
             keyboard: KEYBOARD_AZERTY,
-            today: moment(),
+            today: moment('2022-03-08T00:00:00'),
```

Comment:
Et enfin, pour ne pas oublier de le retirer avant de merge. ;)
```suggestion
            today: moment(),
```